### PR TITLE
Add live test mode

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -47,12 +49,18 @@ sanity: $(MANIFEST) | tests/playbooks/vars/server.yaml
 test: $(MANIFEST) | tests/playbooks/vars/server.yaml
 	$(PYTEST) $(TEST)
 
+livetest: $(MANIFEST) | tests/playbooks/vars/server.yaml
+	pytest -v 'tests/test_playbooks.py::test_playbook' --vcrmode live
```
Comment (mdellweg) [inline Makefile:53]: Here the tests must be run sequentially.

File: tests/conftest.py [hunk 2]
```diff
@@ -3,10 +3,15 @@
 
 def pytest_addoption(parser):
     parser.addoption(
-        "--record", action="store_true", help="record new server-responses"
+        "--vcrmode",
+        action="store",
+        default="replay",
+        help="mode for vcr recording; one of ['replay', 'record', 'live']",
```
Comment (evgeni) [inline tests/conftest.py:10]: I think you can use `choices` here like in argparse (https://docs.python.org/3/library/argparse.html#choices, https://docs.pytest.org/en/latest/reference.html#pytest.config.argparsing.Parser.addoption), then you don't need the `assert` later.

**Review by evgeni — CHANGES_REQUESTED** [review]
I like that! One suggestion, good to go after that.

File: tests/conftest.py (hunk 2 above)
Comment (evgeni) [reply tests/conftest.py:10]: Yes, I had to read `pytest` source to figure that out.

Comment (mdellweg) [reply tests/conftest.py:10]: Now that i found the docs, i know that this parser is _from_ argparse. ;)

**Review by evgeni — APPROVED** [review]

**Pull request merged**